test/OpenIddict.Server.Tests/AddServerRegistersDefaultHandlerDataAttribute.cs

```diff
@@ -0,0 +1,21 @@
+using System.Collections.Generic;
+using System.Reflection;
+using Xunit.Sdk;
+
+namespace OpenIddict.Server.Tests
+{
+    public class AddServerRegistersDefaultHandlerDataAttribute : DataAttribute
```

Comment:
Since it's not used in multiple class tests, we should probably avoid this extra attribute and use `[MemberData]` instead (e.g https://github.com/openiddict/openiddict-core/blob/1e87e917a688087d8612a6adab4c203b95f9187f/test/OpenIddict.Abstractions.Tests/Primitives/OpenIddictResponseTests.cs#L113)

Reply:
Removed.